### PR TITLE
BD-2504: Add caveat for being globally unsubscribed

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 1]
```diff
@@ -141,9 +143,9 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
...
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{% alert tip %}
+Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to either send an email with or without an unsubscribe link.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:147]:
```suggestion
Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 2]
```diff
@@ -141,9 +143,9 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
...
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{% alert tip %}
+Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:147]:
```suggestion
Be very careful when using a template with the custom footer {%raw%}``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``{%endraw%} when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 3]
```diff
@@ -141,9 +143,9 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
...
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{% alert tip %}
+Be very careful when using a template with the custom footer {%raw%}``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``{%endraw%} when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:147]:
```suggestion
Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}`` when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 4]
```diff
@@ -141,9 +143,9 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
 2. Go to the **Custom Footer** section and turn on custom footers.
 3. Edit your footer in the **Compose** section and send a test message. <br><br>{% raw %}The default footer uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute and Braze's physical mailing address. To comply with CAN-SPAM regulations, your custom footer must include ``{{${set_user_to_unsubscribed_url}}}``. You won't be able to save a custom footer without this attribute. <br><br> ![][20] <br><br> If using the default footer, which uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute, be sure to select **&#60;other&#62;** for the **Protocol**. <b><br>![Protocol and URL values needed for the custom footer.][24]{: style="max-width:50%;"}
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{% alert tip %}
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:146]:
```suggestion
{%raw%}
{% alert tip %}
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 5]
```diff
@@ -141,9 +143,9 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
...
+Be very careful when using a template with the custom footer {%raw%}``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``{%endraw%} when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
+{% endalert %}
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:148]:
```suggestion
{% endalert %}
{%endraw%}
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 6]
```diff
@@ -141,9 +143,11 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
...
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{%raw%}
+{% alert tip %}
+Be very careful when using a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}`` when composing an email campaign. A warning will appear, but it'll be your choice to send an email with or without an unsubscribe link.
+{% endalert %}
+{%endraw%}
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:150]:
```suggestion
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md [hunk 7]
```diff
@@ -141,9 +143,11 @@ If you are using the [older navigation]({{site.baseurl}}/navigation), this page
 2. Go to the **Custom Footer** section and turn on custom footers.
 3. Edit your footer in the **Compose** section and send a test message. <br><br>{% raw %}The default footer uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute and Braze's physical mailing address. To comply with CAN-SPAM regulations, your custom footer must include ``{{${set_user_to_unsubscribed_url}}}``. You won't be able to save a custom footer without this attribute. <br><br> ![][20] <br><br> If using the default footer, which uses the ``{{${set_user_to_unsubscribed_url}}}`` attribute, be sure to select **&#60;other&#62;** for the **Protocol**. <b><br>![Protocol and URL values needed for the custom footer.][24]{: style="max-width:50%;"}
 
-
-
-> Be very careful to use a template with the custom footer ``{{${email_footer}}}`` or ``{{${set_user_to_unsubscribed_url}}}``when composing an email campaign. A warning will pop up, but it'll be your choice to either send an email with or without an unsubscribe link.
+{%raw%}
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:146]:
```suggestion
```

File: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md (hunk 4 above)
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md:146]: @lydia-xie there's a raw tag above this that is never closed, which seems to be why your alert isn't rendering.

**Review by rachel-feinberg — APPROVED** [review]
Lgtm when the alert is fixed!

**Pull request merged**